From what I can see in the logs, the problem seems to be with the infura connection.

Not sure if this has something to do with a simple one-character typings change.

Do any of the other PRs have the same problem?

```
>>>>>>
HTTP:MAINNET getBlock
>>>>>>
Error: Failed to connect to Infura over websockets after 10 tries
    at getBlockWithRetry (D:\a\web3.js\web3.js\windows_test\basic_usage.js:31:15)
Error: Process completed with exit code 1.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -607,6 +607,7 @@ Released with 1.0.0-beta.37 code base.
 
 -  Fixed types for getPastEvents (#4955) (#5260)
 -  Fix Log type by adding missing `removed` property (#4877)
+-  Fixed types for `web3.utils._jsonInterfaceMethodToString` (#5550)
 
 ### Changed
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -607,7 +607,6 @@ Released with 1.0.0-beta.37 code base.
 
 -  Fixed types for getPastEvents (#4955) (#5260)
 -  Fix Log type by adding missing `removed` property (#4877)
--  Fixed types for `web3.utils._jsonInterfaceMethodToString` (#5550)
 
 ### Changed
 
@@ -616,3 +615,5 @@ Released with 1.0.0-beta.37 code base.
 ## [Unreleased]
 
 ## [1.8.2]
+
+-  Fixed types for `web3.utils._jsonInterfaceMethodToString` (#5550)

--- a/packages/web3-utils/types/index.d.ts
+++ b/packages/web3-utils/types/index.d.ts
@@ -169,7 +169,7 @@ export interface Utils {
     isContractAddressInBloom(bloom: string, contractAddress: string): boolean;
     isTopicInBloom(bloom: string, topic: string): boolean;
     isTopic(topic: string): boolean;
-    jsonInterfaceMethodToString(abiItem: AbiItem): string;
+    _jsonInterfaceMethodToString(abiItem: AbiItem): string;
     soliditySha3(...val: Mixed[]): string | null;
     soliditySha3Raw(...val: Mixed[]): string;
     encodePacked(...val: Mixed[]): string | null;

